src/routes/+layout.svelte

```diff
@@ -152,6 +156,10 @@
 		rel="manifest"
 		href="{PUBLIC_ORIGIN || $page.url.origin}{base}/{PUBLIC_APP_ASSETS}/manifest.json"
 	/>
+
+	{#if PUBLIC_PLAUSIBLE_DOMAIN}
+		<script defer data-domain={PUBLIC_PLAUSIBLE_DOMAIN} src="/js/script.js"></script>
+	{/if}
```

Comment:
```suggestion
	{#if PUBLIC_PLAUSIBLE_SCRIPT_URL}
		<script defer data-domain={new URL(PUBLIC_ORIGIN).hostname} src={PUBLIC_PLAUSIBLE_SCRIPT_URL}></script>
	{/if}
```

something like this IMO, so other users of HF chat don't have to have a proxy

Reply:
Ah that's fair thanks, will update the variables for this